main.go

```diff
@@ -136,13 +136,10 @@ func initFlags(fs *pflag.FlagSet) {
 
 func validateFlags() error {
 	switch options.ProviderIDFormatType(options.ProviderIDFormat) {
-	// Deprecated: ProviderIDFormatV1 is deprecated and will be removed in a future release.
-	case options.ProviderIDFormatV1:
-		setupLog.Info("Using v1 version of ProviderID format.V1 is deprecated and will be removed in a future release.Instead use V2.")
 	case options.ProviderIDFormatV2:
```

Comment:
Since we don't have many options here, maybe we can simplify this function to use simple if-else to validate the flags instead of switch case, wdyt?

Reply:
Fixed it.